Library/Formula/cgdb.rb

```diff
@@ -18,9 +18,14 @@ class Cgdb < Formula
     depends_on "automake" => :build
   end
 
+  if OS.linux?
+    depends_on "flex" => :build
+  end
   depends_on "help2man" => :build
   depends_on "readline"
 
+  patch :DATA
```

Comment:
In that case, just ask the upstream authors if they'd be willing to tag a new stable release.